CurrencyConverterControl/Themes/Generic.xaml

```diff
@@ -0,0 +1,75 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="clr-namespace:CurrencyConverterControl">    
+    <Style TargetType="{x:Type local:CurrencyConverterControl}">
+        <Setter Property="Background" Value="{x:Static SystemColors.WindowBrush}" />
+        <Setter Property="Foreground" Value="{x:Static SystemColors.ControlTextBrush}" />
```

Comment:
we always try to avoid setting Foreground in control styles. It must be inheritable. The idea is if someone sets Foreground on the Window level, it should be inherited by all controls inside. Such setter breaks inheritance. I know, Microsoft does the same in many cases, but it's really bad practice. It requires a lot of effort to make foreground properly working with theming and DataTemplates.